sorc/build_workflow_utils.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -eux
+
+[[ $(uname -s) == Darwin ]] && cmd=$(which greadlink) || cmd=$(which readlink)
+readonly UTILS_DIR=$(cd "$(dirname "$($cmd -f -n "${BASH_SOURCE[0]}" )" )" && pwd -P)
+
+# Adapt for global-workflow structure.
+target=${target:-"NULL"}
+modulefile=${UTILS_DIR}/../modulefiles/workflow_utils.$target
+if [[ -f $modulefile ]]; then
+  set +x
+  source ${UTILS_DIR}/machine-setup.sh > /dev/null 2>&1
+  source $modulefile
+  module list
+  set -x
+fi
+# End adaptation
+
+# Begin hack
+# In place until nceplibs-ncio is in hpc-stack and available as a module
+# After nceplibs-ncio is in hpc-stack, add the following line to
+# ${UTILS_DIR}/../modulefiles/workflow_utils.<platform>
+# "module load ncio/<ncio-version>"
+# and remove this hack
```

Comment:
it is an unofficial release.  Ed is working on adding documentation and tests.  When it will be released, we can remove the hack.
This will ensure we work with the current develop leading up to the release.

Reply:
Got it.